### ISSUE TITLE
rendering is not smooth  in scrcpy on 60fps  . i have noticed .. motion blur is not there mirroring

### ISSUE THREAD

**Issue by StylishBoyGaming** [Genymobile/scrcpy#1077]

**Comment by rom1v** [comment]
If you record, then play, is it better?

If you show the FPS in the console (press Ctrl+i in the scrcpy window, look at the logs in the console), what happens? (move things randomly on the screen so that the screen keeps updating at a maximum frame rate)

**Comment by StylishBoyGaming** [comment]
@rom1v  recording not have these problem... but this problem is with live time rendering .. try to to understand there is with your renderer 
https://github.com/barry-ran/QtScrcpy @rom1v use this modded scrcpy ..you will understand my point. this modded scrcpy is smoother than original scrcpy .. plzz understand my point

**Comment by rom1v** [comment]
Which version of scrcpy do you use?

Try with a different version of SDL (see #425).

**Comment by StylishBoyGaming** [comment]
@rom1v i use 1.12.1 scrcpy .  i am talking about g sync and smoothness 
in scrcpy .. hope you will understand ..

**Comment by StylishBoyGaming** [comment]
@rom1v  use this  https://github.com/barry-ran/QtScrcpy ... and play any 60 fps game in mobile you will understand ..plz its will improve you scrcpy project.

**Comment by StylishBoyGaming** [comment]
https://github.com/barry-ran/QtScrcpy 
Can you use this opengl renderer instead of sdl renderer plzzz ..?

**Comment by rom1v** [comment]
SDL uses OpenGL on Linux, Direct3d on Windows, etc.

https://wiki.libsdl.org/SDL_HINT_RENDER_DRIVER

Which platform do you use?

Could you please enable FPS loggkng in the console: https://github.com/Genymobile/scrcpy/issues/1077#issuecomment-573334056

**Comment by StylishBoyGaming** [comment]
@rom1v  i use windows 10 pro

**Comment by rom1v** [comment]
Try to force opengl by setting the hint in the code: https://wiki.libsdl.org/SDL_HINT_RENDER_DRIVER

Maybe your direct3d driver is slower.

**Comment by StylishBoyGaming** [comment]
@rom1v  how to force sdl to render through opengl in windows ..can you do this for me plzz i dont have much knowledge about codding

**Comment by dilbert168** [comment]
@rom1v I also has this issue while playing a test video clip at 60 fps, I noticed that frame drop on Window10 with i5 PC connecting with USB.

I understand that Android Q together with v1.12.1 could change the fps on encoder side.
However, my project hardware is fixed at Android-9 and cannot be changed.
Will it be possible to change the code on Server side myself to fixed at 30fps ?

Thanks so much.

**Comment by rom1v** [comment]
@dilbert168 yes, https://github.com/Genymobile/scrcpy/issues/488#issuecomment-567439415

**Comment by RajeevAvr** [comment]
Can i use a type-c (android)to type-c (laptop) to use scrcpy

**Comment by rom1v** [comment]
Probably, as soon as `adb` detects your device, it should work.
